lib/datapoints.js

```diff
@@ -204,13 +207,16 @@ const deviceTypes = {
     shellyplusuni: ['shellyplusuni'],
     ShellyWallDisplay: ['ShellyWallDisplay'],
     shellyplusrgbwpm: ['shellyplusrgbwpm'],
-    // Gen 3
+     // Gen 3
     shelly1minig3: ['shelly1minig3'],
     shelly1pmminig3: ['shelly1pmminig3'],
     shellypmminig3: ['shellypmminig3'],
     shellyhtg3: ['shellyhtg3'],
     shelly1pmg3: ['shelly1pmg3'],
     shelly1g3: ['shelly1g3'],
+    shelly2pmg3: ['shelly2pmgen3'], 
+
+    
 };
```

Comment:
Fix lint errors

Reply:
Renamed shelly2pmgen3 to shelly2pmg3 to be consistent with the naming scheme

Reply:
Not sure if linter ran sucessfull

Reply:
If you are not sure, please rerun linter :-)
`npm run lint` (at your dev directory) should do it

Reply:
runs through without errors